docs/resources/service_pan.md

```diff
@@ -103,11 +118,13 @@ resource "alkira_service_pan" "test1" {
 
 Required:
 
-- `credential_id` (String) ID of PAN instance credential managed by credential resource.
 - `name` (String) The name of the PAN instance.
 
 Optional:
 
+- `auth_code` (String) PAN instance auth code. Only required when `license_type` is `BRING_YOUR_OWN`.
+- `auth_key` (String) PAN instance auth key. This is only required when `panorama_enabled` is set to `true`.
+- `credential_id` (String) ID of PAN instance credential.
```

Comment:
I thought credential_id will be in the Read-Only section... weird why it shows up in the Optional